api/errors/notFoundError.js

```diff
@@ -0,0 +1,10 @@
+const CustomErrorType = require('./customErrorType');
+
+class NotFoundError extends Error {
+  constructor(message) {
```

Comment:
It might be reasonable for the constructor to input a resource type (e.g. 'Player', 'Game', and then an id and programmatically construct the message for consistent formatting, but this is easy and flexible